modules/compliance-tests/src/smithy4s/compliancetests/internals/MappedHintsSchemaVisitor.scala

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.compliancetests.internals
+
+import smithy.api._
+import smithy4s.schema.{
+  Alt,
+  CollectionTag,
+  EnumValue,
+  Primitive,
+  Schema,
+  SchemaAlt,
+  SchemaField,
+  SchemaVisitor
+}
+import smithy4s.{kinds, Bijection, HintMask, Hints, Lazy, Refinement, ShapeId}
+import smithy4s.compliancetests.internals.MappedHintsSchemaVisitor.hintMask
+import smithy4s.schema.Schema._
+
+object MappedHintsSchemaVisitor {
```

Comment:
I think we could do this more concisely. Remind me the problem this solves again ?

Reply:
Alright, I remember ... I'm started to think the better course of action would be to align on what AWS does, and change the behaviour for 0.18. 

The maintenance burden induced by the difference is starting to be somewhat uncomfortable. I think aligning would be preferable, for our own sake, as well as for the sake of the ecosystem, considering the difference of default violates the principle of least-surprise ... 

In order to reduce the risk based on the change of behaviour in user's applications, we could add a smithy-level validator (in alloy) that would issue a warning for any timestamp shape connected transitively to a service with `alloy#simpleRestJson` stating that it's preferable for users to be explicit about timestamp formats ... 

What do you think ?

Reply:
@yisraelU, I've created the `series/0.18`. When you have the time, could you : 

- [ ] Change this PR to target that new series branch
- [ ] Sync this PR with the new target 
- [ ] Change the default timestamp format to EPOCH_SECONDS in the relevant Json/Document schema visitors 
- [ ] Remove the then un-necessary Service/Endpoint transformation 

Thank you in advance ;)